servers/Azure.Mcp.Server/docs/e2eTestPrompts.md

```diff
@@ -625,6 +625,40 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | storage_blob_get | Show me the blobs in the blob container <container> in the storage account <account> |
 | storage_blob_upload | Upload file <local-file-path> to storage blob <blob> in container <container> in storage account <account> |
 
+## Azure Storage Sync
+
+| Tool Name | Test Prompt |
+|:----------|:----------|
+| storagesync_service_create | Create a new Storage Sync Service named <service-name> in resource group <resource-group-name> at location <location> |
+| storagesync_service_delete | Delete the Storage Sync Service <service-name> from resource group <resource-group-name> |
+| storagesync_service_get | Get the details of Storage Sync Service <service-name> in resource group <resource-group-name> |
+| storagesync_service_get | List all Storage Sync Services in resource group <resource-group-name> |
+| storagesync_service_list | List all Storage Sync Services in my subscription |
```

Comment:
This tool was merged with get, so we need to remove all storagesync_service_list tools from this doc.